contracts/Controller.sol

```diff
@@ -267,7 +329,7 @@ contract Controller is Initializable, OwnableUpgradeSafe, ReentrancyGuardUpgrade
      * @dev can only be called when system is not paused
      * @param _actions array of actions arguments
      */
-    function operate(Actions.ActionArgs[] memory _actions) external payable nonReentrant {
+    function operate(Actions.ActionArgs[] memory _actions) external payable nonReentrant notShutdown {
```

Comment:
Just to verify my understanding:

1. If the system is paused, mint, short, burn, add collateral (all actions except exercise / settleVault are paused
2. if the system is shutdown, all actions are stopped

Reply:
Yeah right.